frontend/src/pages/application/createInstance.vue

```diff
@@ -68,7 +69,7 @@ export default {
         NavItem,
         SideNavigation
     },
-    mixins: [applicationMixin],
+    mixins: [applicationMixin, permissionMixin],
```

Comment:
I can't see where this is used in this component? We use the `team`, but that's already imported in the `mapState` directly, so need for the mixin here.

Reply:
Good catch! It doesn't seem to be used anywhere